src/thrift_parser.yrl

```diff
@@ -159,8 +159,8 @@ Throws -> throws '(' FieldList ')': '$3'.
 FieldList -> '$empty': [].
 FieldList -> Field FieldList: ['$1'|'$2'].
 
-Field -> FieldIdentifier FieldRequired FieldType ident FieldDefault Annotations Separator:
-    build_node('Field', line('$4'), '$6', ['$1', '$2', '$3', unwrap('$4'), '$5']).
+Field -> FieldIdentifier FieldRequired FieldType Annotations ident FieldDefault Annotations Separator:
```

Comment:
Annotation following the `FieldType` is supported by apache and fbthrift. This can be used to signal the type of map/set/list.